File: csla20vb/Csla/DefaultFilter.vb
Imports System
Imports System.Collections.Generic
Imports System.Text

Friend Class DefaultFilter

  Public Shared Function Filter(ByVal item As Object, ByVal filterValue As Object) As Boolean

    Dim result As Boolean = False

    If Not item Is Nothing AndAlso Not filterValue Is Nothing Then
      result = CStr(item).Contains(CStr(filterValue))
    End If

    Return result

  End Function

End Class

File: csla20cs/Csla/DefaultFilter.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Csla
{
  internal static class DefaultFilter
  {
    public static bool Filter(object item, object filter)
    {
      bool result = false;
      if (item != null && filter != null)
        result = item.ToString().Contains(filter.ToString());
      return result;
    }
  }
}
